docs/releases/overview.md

```diff
@@ -18,7 +18,39 @@ See the full release notes on [GitHub ↗](https://github.com/vegaprotocol/vega/
+
+
+### Improvements
+
+- A spec update has been made to not update the book price during an auction. The new functionality now mean that when a futures, or perpetual, market is in a monitoring auction, book price is undefined with staleness increasing with time, the book price at auction uncrossing should be set to the price of the uncrossing trade, the mark price should only be recalculated when the auction exits, starting from only the last period indicated by `network.markPriceUpdateMaximumFrequency`. This was introduced in [issue 10810 ↗](https://github.com/vegaprotocol/vega/issues/10810).
+- The price monitoring engine has now been updated to be in-line with new mark-price-methodology. The price monitoring engine now tracks mark price evolution within the market. It is used to monitor trades as well as mark price update candidates. If any of these violates the bounds the market should go into auction and the transaction should get rejected. This was introduced in the following issues: [issue 10845 ↗](https://github.com/vegaprotocol/vega/issues/10845) and [issue 10392 ↗](https://github.com/vegaprotocol/vega/issues/10392).
+- The margin calculations have now been simplified. This has been done by removing the orderbook-based exit price calculation from margin calculation. This was carried out in [issue 10754 ↗](https://github.com/vegaprotocol/vega/issues/10754).
+- The validation for lower bound for margin factor has been updated. The protocol now allows, when switching to isolated margin mode, a valid value of the margin factor to be greater than `0`, but also greater than `max(risk factor long, risk factor short) + linear slippage factor`. This has been updated in [issue 10846 ↗](https://github.com/vegaprotocol/vega/issues/10846).
+- A new API has been created to expose the `notionalTimeWeightedAveragePosition` of a party. This is so that front end dApps can compare to the `notionalTimeWeightedAveragePositionRequirement` on the transfers API to see if they are eligible for rewards. THis has been added in [issue 10831 ↗](https://github.com/vegaprotocol/vega/issues/10831).
+- The `ethcall` engine will now send to core a dummy chain event if it hasn't sent anything to core in the past hour. It will contain just the last `eth-block-height` checked. This allows core to store in the snapshot a more up to date `last-seen` block height meaning that a node will "re-check" fewer ethereum blocks after a protocol upgrade. This will reduce RPC calls to Ethereum after periods of inactivity sourcing data if there are no markets sourcing data from Ethereum data sources. This improvement has bee made in [issue 10841 ↗](https://github.com/vegaprotocol/vega/issues/10841).
```

Comment:
```suggestion
- The `ethcall` engine will now send to core a dummy chain event if it hasn't sent anything to core in the past hour. It will contain just the last `eth-block-height` checked. This allows core to store in the snapshot a more up to date `last-seen` block height meaning that a node will "re-check" fewer ethereum blocks after a protocol upgrade. This will reduce RPC calls to Ethereum after periods of inactivity sourcing data if there are no markets sourcing data from Ethereum data sources. This improvement has been made in [issue 10841 ↗](https://github.com/vegaprotocol/vega/issues/10841).
```